docs/getting-started/integrations-cdn/react-default-cdn.md

```diff
@@ -38,7 +38,228 @@ You get ready-to-use code tailored to your needs!
 
-## Component properties
+### Installing react component from npm
+
+Install the `@ckeditor/ckeditor5-react` package:
+
+```bash
+npm install @ckeditor/ckeditor5-react
+```
+
+The `useCKEditorCloud` hook is responsible for returning information that:
+
+- CKEditor is still being downloaded from the CDN with status = "loading".
+- An error occurred during the download when status = "error", then further information is returned in the error field.
+- Returning the editor in the data field and its dependencies when status = "success".
+
+Use the `<CKEditor>` component inside your project. The below example shows how to use the component with the open-source plugins.
+
+```js
+import React from "react";
+import { CKEditor, useCKEditorCloud } from "@ckeditor/ckeditor5-react";
+
+const CKEditorDemo = () => {
+	const cloud = useCKEditorCloud( {
+		version: "43.0.0",
+	} );
+
+	if ( cloud.status === "error" ) {
+		return <div>Error!</div>;
+	}
+
+	if ( cloud.status === "loading" ) {
+		eturn <div>Loading...</div>;
+	}
+
+	const {
+		ClassicEditor,
+		Bold,
+		Essentials,
+		Italic,
+		Paragraph,
+		Undo,
+	} = cloud.CKEditor;
+
+	return (
+		<CKEditor
+			editor={ClassicEditor}
+			data={"<p>Hello world!!!</p>"}
+			config={{
+			toolbar: {
+				items: ["undo", "redo", "|", "bold", "italic"],
+			},
+			plugins: [ Bold, Essentials, Italic, Paragraph, Undo ],
+			}}
+		/>
+	);
+};
+```
+
+To use premium plugins, set the `premium` property to `true` in the `useCKEditorCloud` configuration and provide your license key in the `CKEditor` configuration.
+
+```js
+import React from "react";
+import { CKEditor, useCKEditorCloud } from "@ckeditor/ckeditor5-react";
+
+const CKEditorDemo = () => {
+	const cloud = useCKEditorCloud({
+		version: "43.0.0",
+		premium: true,
+	});
+
+	if ( cloud.status === "error" ) {
+		return <div>Error!</div>;
+	}
+
+	if ( cloud.status === "loading" ) {
+		return <div>Loading...</div>;
+	}
+
+	const {
+		ClassicEditor,
+		Bold,
+		Essentials,
+		Italic,
+		Mention,
+		Paragraph,
+		Undo,
+	} = cloud.CKEditor;
+
+	const { SlashCommand } = cloud.CKEditorPremiumFeatures!;
+
+	return (
+		<CKEditor
+			editor={ ClassicEditor }
+			data={ "<p>Hello world!!!</p>" }
+			config={{
+				licenseKey: "<YOUR_LICENSE_KEY>",
+				toolbar: {
+					items: [ "undo", "redo", "|", "bold", "italic" ],
+				},
+				plugins: [
+					Bold,
+					Essentials,
+					Italic,
+					Mention,
+					Paragraph,
+					Undo,
+					SlashCommand,
+				],
+			}}
+		/>
+	);
+};
+```
+
+### Usage with CKBox
+
+To use `CKBox`, specify the version and theme (optionally) in the `useCKEditorCloud` configuration. Also, remember about the actual plugin configuration inside `<CKEditor/>` component.
+
+```js
+import React from "react";
+import { CKEditor, useCKEditorCloud } from "@ckeditor/ckeditor5-react";
+
+const CKEditorDemo = () => {
+	const cloud = useCKEditorCloud({
+	version: "43.0.0",
+	ckbox: {
+		version: "2.5.1",
```

Comment:
Will `"latest"` be also a proper value? :thinking:

Reply:
At this moment - no. Imho, latest should be avoided to protect integration against breaking changes.